substrate/frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -76,10 +76,6 @@ pub fn expand_outer_config(
 			#fields
 		}
 
-		#[cfg(any(feature = "std", test))]
-		#[deprecated(note = "GenesisConfig is planned to be removed in December 2023. Use `RuntimeGenesisConfig` instead.")]
-		pub type GenesisConfig = RuntimeGenesisConfig;
```

Comment:
if you see in FRAME, there are a few more deprecated items that we can already remove. You do them in a follow up if interested :) Assuming min 6mo has passed since being marked deprecated